### ISSUE TITLE
Login credentials check

### ISSUE THREAD

**Issue by ttouf** [troydavisson/PHRETS#292]
Is there a way to check if the username and password are correct?
I'm using 2.6.1

**Comment by troydavisson** [comment]
https://retsmd.com/auth

**Comment by ttouf** [comment]
Thanks for your quick reply...Can I use php to check it?

**Comment by fabcom-aaron** [comment]
I am receiving a 500 status code when testing at the retsmd auth url?

Thanks in advance.

**Comment by troydavisson** [comment]
Yeah, it broke during a recent server upgrade.  I'll try to patch it up soon.
